kclvm/tools/src/LSP/src/quick_fix.rs

```diff
@@ -40,6 +40,30 @@ pub(crate) fn quick_fix(uri: &Url, diags: &Vec<Diagnostic>) -> Vec<lsp_types::Co
                                 }));
                             }
                         }
+                        ErrorKind::InvalidSyntax => {
+                            let replacement_texts = extract_suggested_replacements(&diag.data);
+                            for replacement_text in replacement_texts {
+                                let mut changes = HashMap::new();
+                                changes.insert(
+                                    uri.clone(),
+                                    vec![TextEdit {
+                                        range: diag.range,
+                                        new_text: replacement_text.clone(),
+                                    }],
+                                );
+                                code_actions.push(CodeActionOrCommand::CodeAction(CodeAction {
+                                    title: "Split multiple assignment into separate assignments"
```

Comment:
This error message doesn't seem to be related to the code in its vicinity. Here, how do we determine that the illegal error must be a multiple assignment.